src/commands/lint/configureLintingToolsCommand.ts

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { commands, l10n, window, ExtensionContext } from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import { WorkspaceUtils } from '../../utils/workspaceUtils';
+import { TAB_SPACES } from '../../utils/constants';
+
+const configureLintingToolsCommand =
+    'salesforcedx-vscode-offline-app.configureLintingTools';
+const eslintDependencies = [
+    ['@salesforce/eslint-plugin-lwc-graph-analyzer', '^0.9.0'],
```

Comment:
These are configuration items where at least the version values would be better represented in [contributes.configuration](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration) settings. That way a) we take hard-coded data out of the code, and b) users can update these values in their own personal configuration settings, to override the versions we select.

Reply:
![image](https://github.com/salesforce/salesforcedx-vscode-mobile/assets/42476999/3637c6b4-d6a2-41e2-8c22-a4a948a05cf1)